CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 
 ### Removed
 - Removed possibility to export entries/databases to an `.sql` file, as the logic cannot easily use the correct escape logic
+- Removed support of old groups format (used prior to JabRef version 1.6)
```

Comment:
No this is not true. With this PR exactly this migration code is removed, so groups in 10 years old bib files are not read correctly. But every version between 1.7 and 3.3 converted the old format to the current one.